csrc/multidevice/lower_communication.cpp

```diff
@@ -397,10 +391,19 @@ void lowerToReduceScatter(
   params.redOp = getC10dReduceOpType(op_type);
   params.team = mesh.vector();
   params.dst_bufs = {output_tensor};
+  // TODO: Clean up once allocation domain for sharded tensors is
+  // better supported.
+  auto red_axis = output_tv->getReductionAxis().value();
```

Comment:
```suggestion
  auto reduction_axis = output_tv->getReductionAxis().value();
```

The typing saved isn't worth the confusion.